internals/types/index.d.ts

```diff
@@ -42,12 +67,6 @@ export interface GlobalConfig {
   telemetry?: {
     enabled?: boolean;
   };
-
-  // TODO: legacy - remove
-  updateChannel?: string;
-  desktop?: {
-    teamOrder: any;
-  };
```

Comment:
I found no reference to `updateChannel`, and `desktop` was only referenced in `logout`, where it just got deleted.